go.mod

```diff
@@ -12,13 +13,15 @@ require (
 	github.com/hashicorp/go-retryablehttp v0.7.2
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.0
-	github.com/okta/okta-sdk-golang/v2 v2.17.0
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/okta/okta-sdk-golang/v3 v3.0.1
```

Comment:
I would prefer that we do an update on okta sdk golang v3.0.2 to not have it generate `Get` on all the getter names as we discussed. As the v3 hasn't be been marked as the latest release over the v2 series on the GH releases setting I think it is ok to have a large change of method names on a point release.

Reply:
See above comment